server/models/Event.js

```diff
@@ -13,9 +13,14 @@ const eventSchema = new mongoose.Schema({
     type: Date,
     required: true
   },
-  // eventTimeZone:{
-  //   type: Date,
-  // }
+  eventLink:{
+    type:String,
+    required: true
+  },
```

Comment:
This would be problematic if a user bookmarked an event and that event gets changed in the future, it would be a poor user experience. Could we refactor to utilize route params more?